Makefile

```diff
@@ -268,9 +268,16 @@ $(APIDOCS_TARGET_DIR): | $(DIST_DIR)
 $(APIDOCS_ZIP_FILE): $(APIDOCS_TARGET_DIR)
 	$(BASHLIKE_SHELL_COMMAND) "cd $(DIST_DIR) && $(ZIP) -r $(notdir $(APIDOCS_ZIP_FILE)) $(notdir $(APIDOCS_TARGET_DIR))"
 
+# need to get the working directory path because ln doesn't work with relative paths
+WORKING_DIR := $(shell pwd)
+ifeq ($(OS),Windows_NT)
+	# this setting is necessary for ln to work on git bash for windows
+	export MSYS = winsymlinks:nativestrict
```

Comment:
I'm slightly surprised this is working with the spaces around `=` because I think that treats `MSYS` as a command and `=` as it's first argument. https://unix.stackexchange.com/questions/258727/spaces-in-variable-assignments-in-shell-scripts

Reply:
Is it possible this block is somehow not running and the symlink is still working?

Reply:
I just tried commenting out that line in ee01dca and it once again fails with "No space left on device": https://github.com/natcap/invest/pull/571/checks?check_run_id=2858930559#step:8:675 In 9991fa8 I un-commented it and called `echo $(MSYS)` and it is the expected value: https://github.com/natcap/invest/runs/2859138132#step:8:666

[This page](https://www.gnu.org/software/make/manual/html_node/Variables_002fRecursion.html) says _"To pass down, or export, a variable, make adds the variable and its value to the environment for running each line of the recipe."_ shows the `export variable = value` syntax. That's in the context of setting env variables for sub-make commands, but I think it's the same. [Here's](https://stackoverflow.com/a/27282778/14451410) a stackoverflow answer suggesting the same thing for making a variable available to commands in a make recipe.

Reply:
I'm guessing it's different from what that unix.stackexchange post says because it's being handled by gnu make, not bash.

Reply:
Ah thanks for the `make` lesson :)